Source/GMCAbilitySystem/Public/Ability/GMCAbility.h

```diff
@@ -78,6 +78,14 @@ class GMCABILITYSYSTEM_API UGMCAbility : public UObject, public IGameplayTaskOwn
 	UFUNCTION(BlueprintPure, Category="GMCAbilitySystem|Ability")
 	AActor* GetOwnerActor() const;
 
+	/** Get the Pawn associated with ability if applicable. */
+	UFUNCTION(BlueprintPure, Category="GMCAbilitySystem|Ability")
+	APawn* GetOwnerPawn() const;
+
+	/** Get the Player Controller associated with the owning pawn if applicable. */
+	UFUNCTION(BlueprintPure, Category="GMCAbilitySystem|Ability")
+	APlayerController* GetOwningPlayerController() const;
```

Comment:
Should this return AGMC_PlayerController?

Reply:
all set